### PR TITLE
Fix for adblockers blocking the Quantcast case study page and images

### LINKED ISSUE #292: Quantcast case study 404s when uBlock Origin is enabled
When the popular uBlock Origin ad-blocking Chrome extension is enabled, https://about.sourcegraph.com/case-studies/quantcast/ flashes the use case content but then shows:

image

Disabling uBlock Origin fixes this problem. It seems like the presence of `quantcast` in the URL causes uBlock Origin to think that the page data is an ad (so it is blocked).

Suggested fix: rename the quantcast case study page to not include `quantcast` in the URL, and set up redirects.

### PR DESCRIPTION
**Pull request by ryan-blunden**
uBlock origin (and presumably other adblockers) were blocking Quantcast images and the case study page itself. This has been fixed by renaming the cast study page and associate images.

As a result, requests to /case-studies/quantcast are redirected to /case-studies/quantcast-large-scale-refactoring.

Fixes #292

### CODE REVIEW

File: blogposts/quantcast-large-scale-refactoring-with-sourcegraph.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ tags: [
   blog
 ]
 slug: quantcast-large-scale-refactoring-with-sourcegraph
-heroImage: https://about.sourcegraph.com/case-studies/quantcast_thumbnail_500x500px-generic.jpg
+heroImage: http://localhost:8000/case-studies/quantcast-sourcegraph-case-study.jpg
```
Comment (dadlerj) [inline blogposts/quantcast-large-scale-refactoring-with-sourcegraph.md]:
```suggestion
heroImage: https://about.sourcegraph.com/case-studies/quantcast-sourcegraph-case-study.jpg
```

File: blogposts/quantcast-large-scale-refactoring-with-sourcegraph.md [hunk 2]
```diff
@@ -21,4 +21,4 @@ Founded in 2006, Quantcast's engineering team had amassed thousands of repositor
 
 Learn how, Sourcegraph's [multi-repository code search](/product/code-search-navigation) makes [large scale refactoring at Quantcast](/case-studies/quantcast) systematic, safe, and efficient.
 
-<a href="/case-studies/quantcast" class="btn btn-primary mt-4">Learn more</a>
+<a href="/case-studies/quantcast-large-scale-refactoring" class="btn btn-primary mt-4">Learn more</a>
```
Comment (dadlerj) [inline blogposts/quantcast-large-scale-refactoring-with-sourcegraph.md]: Did you confirm that this fixes it? Just want to be sure it isn't just checking for quantcast _anywhere_ in the name...

Comment (ryan-blunden) [reply blogposts/quantcast-large-scale-refactoring-with-sourcegraph.md]: Yes. I checked locally and confirmed the page and images are loading at the preview site - https://deploy-preview-293--sourcegraph.netlify.com/

File: blogposts/quantcast-large-scale-refactoring-with-sourcegraph.md (hunk 1 above)
Comment (ryan-blunden) [reply blogposts/quantcast-large-scale-refactoring-with-sourcegraph.md]: Thanks, that's been fixed

**Pull request merged**